indico/modules/events/export.yaml

```diff
@@ -221,6 +221,12 @@ export:
     fks_out:
       - event_role_id
 
+  MenuEntryPrincipal:
+    skipif: ROW.type.name not in ('user', 'event_role', 'registration_form')
```

Comment:
And table row matching this condition won't be exported. This is done to get rid of anything that is not portable between instances (category roles and groups) - only events and users are included in exports.